.github/workflows/deploy.yml

```diff
@@ -6,8 +6,8 @@ on:
     branches:
       - main
   pull_request:
-    paths:
-      - "deploy/**"
+  #  paths:
```

Comment:
What's the purpose of this?

Reply:
I'm trying to understand why the deployment is failing. Forcing the execution of this workflow always for testing for the time being. Should be put back as before as soon as it starts working

Reply:
Back as usual. Should be working now